examples/_includes/macros.nj

```diff
@@ -0,0 +1,16 @@
+{% macro description(x) %}
+<text style="description">{{ x }}</text>
+{% endmacro %}
+
+{% macro button(text, buttonProps={}) %}
+<view
```

Comment:
everywhere we call `button`, we also specify `style="button"`, so I'm thinking this should be made default here, when `style` is not already specified in the params?